builtin/providers/aws/resource_aws_cloudwatch_metric_alarm.go

```diff
@@ -95,6 +96,12 @@ func resourceAwsCloudWatchMetricAlarm() *schema.Resource {
 				Optional:      true,
 				ConflictsWith: []string{"statistic"},
 			},
+			"treat_missing_data": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
```

Comment:
The default in web console is also "missing" so I think it make sense to keep "missing"